bot/commands.js

```diff
@@ -156,7 +156,7 @@ const addInvoice = async (ctx, bot, order) => {
 
     if(buyer.lightning_address){
       let laRes = await resolvLightningAddress(buyer.lightning_address,order.amount*1000);
-      if(laRes){
+      if(!(laRes && laRes.pr)){
```

Comment:
Si recibes laRes = undefined `laRes && laRes.pr` esto te va a tirar una excepción porque no puedes usar .pr de undefined

Reply:
pero por el orden de evaluacion se va a evaluar primero laRes, y solo si no es falso, se va a evaluar laRes.pr

Reply:
![imagen](https://user-images.githubusercontent.com/2509615/148830842-05430b7e-d2ee-45e9-b9cc-d1bb473062b1.png)

Reply:
you are goddamn right!